lib/server/methods.coffee

```diff
@@ -92,7 +92,7 @@ Meteor.methods
 	adminAddUserToRole: (_id,role)->
 		check arguments, [Match.Any]
 		if Roles.userIsInRole this.userId, ['admin']
-			Roles.addUsersToRoles _id, role
+			Roles.addUsersToRoles _id, role, Roles.GLOBAL_GROUP
 
 	adminRemoveUserToRole: (_id,role)->
```

Comment:
``adminRemoveUserToRole`` should also use ``Roles.GLOBAL_GROUP`` to be consistent.